apps/web/cypress/plugins/index.ts

```diff
@@ -138,5 +140,62 @@ module.exports = (on, config) => {
         templates,
       };
     },
+    async makeBlueprints() {
+      const dal = new DalService();
+      await dal.connect('mongodb://localhost:27017/novu-test');
+
+      const userService = new UserService();
+      const user = await userService.createUser();
+
+      const notificationGroupRepository = new NotificationGroupRepository();
+      const organizationService = new OrganizationService();
+      const environmentService = new EnvironmentService();
+
+      let organization = await organizationService.getOrganization(config.env.BLUEPRINT_CREATOR);
+      if (!organization) {
+        organization = await organizationService.createOrganization({ _id: config.env.BLUEPRINT_CREATOR });
+        await environmentService.createEnvironment(organization._id, 'Production');
+      }
```

Comment:
I've noticed that for me locally the DB is not cleared after all tests (guess the same for you), so the organization with `BLUEPRINT_CREATOR` id stays there, that's why these checks are needed.

Reply:
I think it is not cleaning the DB on cases where the test run was forced to close. it is a good addon due to the hardcoded id. 🙃